### PR TITLE
#495 feature exception handler

### LINKED ISSUE #495: [FEATURE] Exception handler is not logging any information for the request being made
## Brief description:
As of now, the exception handler will display full traceback but this is not enough when debugging failing requests. 

We need:
- [x] Full url -> this will tell us whether there are any query or path parameters present in the request
- [x] Method type

Additional:
Extract the logic from the health routes into controllers

### PR DESCRIPTION
**Pull request by Aleks1908**
## Added url and method type when there is exception 
Added url and method type to the exception handler and extracted the logic from utility routes to utility controller

resolves #495

### CODE REVIEW

File: packages/services/questionnaire/.svelte-kit/ambient.d.ts
Comment (IvanObreshkov) [inline packages/services/questionnaire/.svelte-kit/ambient.d.ts:1]: This shouldn't be here

**Review by IvanObreshkov — COMMENTED** [review]
Remove svelte files

File: packages/services/questionnaire/.svelte-kit/ambient.d.ts
Comment (Aleks1908) [reply packages/services/questionnaire/.svelte-kit/ambient.d.ts:1]: veche ne e

File: packages/py-api/py_api/routes/utility_routes.py
Comment (IvanObreshkov) [inline packages/py-api/py_api/routes/utility_routes.py:1]: Could you remove logger = getLogger("health")

**Review by IvanObreshkov — APPROVED** [review]
LGTM!

**Pull request merged**